### PR TITLE
add java-version java-distribution cache

### PR DESCRIPTION
**Pull request by AUryadov**
add java-version java-distribution cache to cicd_pipeline_java.yaml and gradle-send-to-sonar

### CODE REVIEW

**Review by realhuseyn — APPROVED** [review]

File: .github/workflows/cicd_pipeline_java.yaml [hunk 1]
```diff
@@ -25,6 +25,20 @@ on:
...
+        description: "Java distribution to use"
+        required: false
+        type: string
+        default: "temurin"
+      java-version:
+        description: "Java version to use"
+        required: false
+        type: string
+        default: "11"
+      cache:
```
Comment (bkbekirov) [inline .github/workflows/cicd_pipeline_java.yaml:38]: `cache` is too general. May be give to this parameter more specific name?

Comment (AUryadov) [reply .github/workflows/cicd_pipeline_java.yaml:38]: java-cache?

File: java-gradle-build/action.yaml [hunk 2]
```diff
@@ -17,6 +17,7 @@ inputs:
   cache:
     description: "Build tool caching"
     required: false
+    default: 'gradle'
```
Comment (bkbekirov) [inline java-gradle-build/action.yaml:20]: What if we send empty string as default value?

Comment (AUryadov) [reply java-gradle-build/action.yaml:20]: sure

**Review by bkbekirov — APPROVED** [review]

**Review by realhuseyn — APPROVED** [review]

**Pull request merged**